### ISSUE TITLE
Update logger dependency

### ISSUE THREAD

**Issue by boskokg** [leisim/logger_flutter#6]
logger_flutter 0.7.1 depends on logger ^0.7.0 and no versions of logger_flutter match >0.7.1 <0.8.0, logger_flutter ^0.7.1 requires logger ^0.7.0.

**Comment by JEuler** [comment]
https://github.com/leisim/logger_flutter/pull/7

**Comment by JEuler** [comment]
You can always do something like:

> logger_flutter:
>     git:
>       url: https://github.com/JEuler/logger_flutter.git

But preferable if you fork it into your own acc, because I can remove it sometime somehow. :)

**Comment by keitoro** [comment]
still the same problem.

**Comment by britannio** [comment]
@leisim

**Comment by voivoed** [comment]
Could @leisim please merge the PR?

**Comment by Cocotus** [comment]
+1 for this, because I'm using flutter web and I need the logconsole to work with the current patched logger [0.9.1](https://pub.dev/packages/logger#-changelog-tab-) ;) At the moment I'm using the workaournd by Jeuler in pubspec.yaml but I think its better to release a new version with the dependency fixed

**Comment by JEuler** [comment]
Yeah, it would be really nice if you release new version, @leisim :) https://github.com/leisim/logger_flutter/pull/8 I create new PR for this, I'm sure you know how to publish update for package :D
But anyway, this will make it easier.

**Comment by Cocotus** [comment]
Well I tried to use newest logger together with current logconsole and I got errors, because a Callback method is not supported in the current logger version.  I dont know how to fix this myself but I think this needs some compatibility work to work with the latest logger version? Can anybody verify this?

**Comment by JEuler** [comment]
Yeah, it is not working for the last version. I've checked. Hm.

**Comment by JEuler** [comment]
So we need to use LogOutput. I will have a look later on weekend days. :)

**Comment by raywhiteside** [comment]
Using the workaround provided, but a gentle +1 that an update would be appreciated. :)

**Comment by tienthanh2509** [comment]
still has dependency error on the official package

**Comment by k415hu** [comment]
Any update on this?

**Comment by g-balas** [comment]
this is working for me : 

logger: 0.7.0+1
logger_flutter: ^0.7.1

**Comment by PoojaB26** [comment]
Hey @leisim any update on this? The issue persists! :)

**Comment by JEuler** [comment]
https://github.com/leisim/logger_flutter/pull/9/files - maybe you can use this PR. 
```
logger_flutter:
git:
	url: https://github.com/ewertonrp/logger_flutter
```
Like this

All credits to @ewertonrp

**Comment by zierka** [comment]
For me this worked:

```
logger_flutter:
    git: https://github.com/ewertonrp/logger_flutter.git
```

**Comment by IanWorthington** [comment]
Seems to break again with the nullsafe version of logger:

```
logger: ^1.0.0-nullsafety.0 #^0.9.4
logger_flutter: #^0.7.1
  git: https://github.com/ewertonrp/logger_flutter.git
```

```
Because every version of logger_flutter from git depends on logger >=0.7.0 <1.0.0 and logging_test depends on logger ^1.0.0-nullsafety.0, logger_flutter from git is forbidden.
So, because logging_test depends on logger_flutter from git, version solving failed.
pub get failed (1; So, because logging_test depends on logger_flutter from git, version solving failed.)
```
